spec/jobs/risc_delivery_job_spec.rb

```diff
@@ -222,10 +222,10 @@
     end
   end
 
-  describe '#warning_error_classes' do
-    it 'when calling warning_error_classes' do
-      expect(described_class.warning_error_classes).to eq(
-        described_class::NETWORK_ERRORS + [ RedisRateLimiter::LimitError ],
+  describe '.warning_error_classes' do
+    it 'is all the network errors and rate limiting errors' do
+      expect(described_class.warning_error_classes).to match_array(
+        [*described_class::NETWORK_ERRORS, RedisRateLimiter::LimitError],
       )
```

Comment:
goals:
1. clarify that it's a class method (`.` vs `#`)
2. use `match_array` for an exact membership check that is also order-independent. If for some reason we refactored `warning_error_classes` to put the rate limiter error first, this spec shouldn't fail, but it should still fail if we removed an error class